api/api.go

```diff
@@ -62,12 +62,37 @@ const (
 	OptTimeoutSec = "TimeoutSec"
 	// OptQuiesceID query parameter use for quiesce
 	OptQuiesceID = "QuiesceID"
+	// OptCredUUID is the UUID of the credential
+	OptCredUUID = "CredUUID"
+	// OptCredType  indicates type of credential
+	OptCredType = "CredType"
+	// OptCredEncrKey indicates if the key is encrypted
+	OptCredEncrKey = "CredEncrypt"
+	// OptCredRegion indicates the region
+	OptCredRegion = "CredRegion"
+	// OptCredDisableSSL indicated if SSL should be disabled
+	OptCredDisableSSL = "CredDisableSSL"
+	// OptCredEndpoint indicate the cloud endpoint
+	OptCredEndpoint = "CredEndpoint"
+	// OptCredAccKey for s3
+	OptCredAccessKey = "CredAccessKey"
+	// OptCredSecretKey
```

Comment:
Could you add some comments on the next few? The comments is only the name of the variable at the moment.

Reply:
Sure. Will add.